tests/sentry/utils/performance_issues/test_consecutive_http_detector.py

```diff
@@ -97,12 +123,101 @@ def test_does_not_detect_consecutive_http_issue_with_frontend_events(self):
         problems = self.find_problems(event)
         assert problems == []
 
-    def test_does_not_detect_consecutive_http_issue_with_low_duration(self):
-        event = self.create_issue_event(100)
-        problems = self.find_problems(event)
+    def test_does_not_detect_consecutive_http_issue_with_low_count(self):
+        spans = [  # count less than threshold
+            create_span("http.client", 20, "GET /api/0/organizations/endpoint1", "hash1"),
+        ]
```

Comment:
Should we add another span and increase it so the time saved is above the new threshold to fully test this is just about the count?

afaik the count threshold could be broken, but this test could still pass because the span is so short against the new time saved threshold

Reply:
My bad, this got left over from when we decreased count to 2. I'll fix this @narsaynorath. Thanks.